src/test/java/ezschedule/testutil/TypicalEvents.java

```diff
@@ -20,7 +20,7 @@ public class TypicalEvents {
     public static final Event CARNIVAL = new EventBuilder().withName("Carnival")
             .withDate("2023-05-03").withStartTime("16:00").withEndTime("18:00").build();
     public static final Event DRAG = new EventBuilder().withName("Drag race")
-            .withDate("2023-05-04").withStartTime("18:00").withEndTime("20:00").build();
+            .withDate("2023-05-03").withStartTime("18:00").withEndTime("20:00").build();
```

Comment:
I think its better NOT to change typical events. Some existing test might rely on the fact that `DRAG`'s date is after `CARNIVAL`'s date. 
Even if it okay during this commit, future tests might depend on this (new) fact that they have the same date. Eventually, (existing and new) tests will depends on different "facts", thus likely to bring around bugs. (When that happens, it would be crazy hard to debug having to dig into commit history).